.github/workflows/cmake.yml

```diff
@@ -688,6 +682,20 @@ jobs:
           Get-Command ccache
           (Get-Command ccache).Source
           ccache --show-stats --verbose --version
+      - name: Download Apache Arrow latest source archive
+        run: |
+          $LATEST_RELEASE_INFO = `
+            Invoke-RestMethod "https://api.github.com/repos/apache/arrow/releases/latest"
+          # e.g. "apache-arrow-19.0.0"
+          $LATEST_TAG_NAME = $LATEST_RELEASE_INFO.tag_name
+          # "apache-arrow-19.0.0" -> "arrow-19.0.0"
+          $ARROW_VERSION = $LATEST_TAG_NAME -replace '^apache-', ''
+          $DOWNLOAD_URL = `
+            "https://www.apache.org/dyn/closer.lua/arrow/${ARROW_VERSION}/${LATEST_TAG_NAME}.tar.gz?action=download"
+          Invoke-WebRequest -Uri $DOWNLOAD_URL -OutFile "${LATEST_TAG_NAME}.tar.gz"
+          mkdir apache-arrow
```

Comment:
How about we unify to "New-Item" command for create directory?
Because we use "New-Item" to create directory in this workflow.

Reply:
fix: [2a16eca](https://github.com/groonga/groonga/pull/2209/commits/2a16eca36a8a0b7b6a72c696b01488ae69476b9a) Sure.